docs/guides/lint-asyncapi.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Lint AsyncAPI
+---
+
+# Lint AsyncAPI with Redocly CLI
+
+In addition to providing lint functionality for multiple OpenAPI formats, Redocly CLI also has support for AsyncAPI.
+Redocly CLI supports the following linting approaches with AsyncAPI documents:
+
+* AsyncAPI document validation, including full binding validation for [supported protocols](#supported-protocols).
+* Supported versions:
+  - AsyncAPI 2.6
+  - earlier versions in the 2.x family may also validate successfully
+* Built-in rules for checking common standards requirements (see the [list of AsyncAPI rules](#asyncapi-rules)).
+* [Configurable rules](../rules/configurable-rules.md) so that you can build your own rules following common patterns
+* [Custom plugins](../custom-plugins/index.md) for advanced users that need additional functionality
+
+## Lint an existing AsyncAPI file
+
+Redocly CLI takes its settings from a `redocly.yaml` configuration file. Below
+is an example of a simple configuration file for validating an AsyncAPI file is
+in the expected format:
+
+```yaml
+extends: []
+
+rules:
+  spec: error
+```
+
+The empty `extends` element instructs Redocly CLI not to use any existing
+rulesets, but to emit an error if the `spec` rule finds any problem. This rule
+checks that the document structure matches what is expected by the AsyncAPI
+specification.
+
+With this configuration file, and your AsyncAPI description file, run the linting command:
+
+```
+redocly lint asyncapi.yaml
+```
+
+The output describes any structural problems with the document, or reports that it is valid.
+
+
+## AsyncAPI rules
+
+To expand the linting checks for an AsyncAPI description, start by enabling
+some of the built-in rules. The currently-supported rules are:
+
+* `info-contact`: the `Info` section must contain a valid `Contact` field.
```

Comment:
Nit: trailing '.' on this bullet, but not on others. Either is fine as long as it's the same throughout the list.

Reply:
Good catch! I am bad at this.